consent/helpers.py

```diff
@@ -19,6 +20,18 @@ def consent_exists(username, course_id, enterprise_customer_uuid):
     :param enterprise_customer_uuid: The consent requester.
     :return: Whether any consent (provided or unprovided) exists.
     """
+    # pylint: disable=invalid-name
+    EnterpriseCourseEnrollment = apps.get_model('enterprise', 'EnterpriseCourseEnrollment')
+    if EnterpriseCourseEnrollment.objects.filter(
+            course_id=course_id,
+            enterprise_customer_user__user_id=get_user_id(username),
```

Comment:
I believe this query will fail if the user doesn't exist and you return `None` from the `get_user_id` function.